integreat_cms/cms/templates/languages/language_form.html

```diff
@@ -57,6 +58,31 @@ <h3 class="font-bold text-black">
                         <div class="mb-2 text-s text-gray-600">{{ form.bcp47_tag.help_text }}</div>
                     </div>
                 </div>
+                {% if form.instance.id %}
+                    <div class="w-full mb-4 rounded border border-solid border-blue-500 shadow-2xl bg-white">
+                        <div class="w-full p-4 rounded bg-water-500">
+                            <h3 class="font-bold text-black">
+                                <i icon-name="gauge" class="mr-2"></i> {% translate "Regions using this language" %}
+                            </h3>
+                        </div>
+                        <div class="w-full p-4">
+                            {% get_language form.instance.slug as language %}
+                            {% get_language_regions language as regions %}
+                            {% get_language_active_regions language as active_regions %}
```

Comment:
This is very picky, but think active_regions as variable name could be misleading and I would rather make it a bit more specific. I hope adding the three more characters doesn't make the variable name too long. 
```suggestion
                            {% get_language_active_regions language as active_in_regions %}
```